src/components/subscription-choices/subscription-choices.twig

```diff
@@ -0,0 +1,61 @@
+{% set _heading_tag = heading_tag|default('h2') %}
+
+<div class="c-subscribe js-subscription-choices{% if class %} {{ class }}{% endif %}">
+  <div class="c-subscribe__controls">
+    <{{_heading_tag}}>
+      {{never_miss_article_heading|default('Never miss an article!')}}
+    </{{_heading_tag}}>
+
+    <div class="c-subscribe__controls-wrapper">
+      {% include '@cloudfour/components/button-swap/button-swap.twig' with {
+        button_class: "c-subscribe__control
+js-subscription-choices__control
+#{notifications_btn_class|default('')}",
+        initial_label: notifications_btn_initial_label|default('Notifications have been turned off.'),
+        initial_visual_label: notifications_btn_initial_visual_label|default('Get notifications'),
+        swapped_label: notifications_btn_swapped_label|default('Notifications have been turned on.'),
+        swapped_visual_label: notifications_btn_swapped_visual_label|default('Turn off notifications'),
+      } only %}
+
+      {% include '@cloudfour/components/button/button.twig' with {
+        href: "#subscription-choices-#{form_id}",
+        class: "c-subscribe__control
+js-subscription-choices__control
+js-subscription-choices__get-weekly-digests-btn
+#{weekly_digests_btn_class|default('')}",
```

Comment:
Nit: In your other examples, you're assembling a class by joining an array. I'm fine with either approach, but we should probably be consistent within the component.

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/2a85120d80af06a4f1be3d8b01206d0f04e786ff